internal/util/sessionutil/session_util.go

```diff
@@ -361,7 +375,7 @@ func (s *Session) registerService() (<-chan *clientv3.LeaseKeepAliveResponse, er
 		}
 		log.Info("put session key into etcd", zap.String("key", completeKey), zap.String("value", string(sessionJSON)))
 
-		keepAliveCtx, keepAliveCancel := context.WithCancel(context.Background())
+		keepAliveCtx, keepAliveCancel := context.WithCancel(s.ctx)
```

Comment:
One concern. In #10981, we change s.ctx to context.Backgroup(). I not sure whether that issue still exist.

Reply:
got it, thanks